Sources/Sentry/SentryThreadInspector.m

```diff
@@ -1,8 +1,14 @@
 #import "SentryThreadInspector.h"
+#import "SentryCrashStackCursor.h"
+#include "SentryCrashStackCursor_MachineContext.h"
+#include "SentryCrashSymbolicator.h"
 #import "SentryFrame.h"
 #import "SentryStacktrace.h"
 #import "SentryStacktraceBuilder.h"
 #import "SentryThread.h"
+#include <pthread.h>
+
+#define MAX_STACKTRACE_LENGTH 100
```

Comment:
`m`: How did we come up with that value?

Reply:
This value was being used here before. I just create the define.

Reply:
Can we then reuse one constant everywhere, please? We can do this in another PR.

Reply:
What to you mean? Instead of `#define` we should use static const?

Reply:
No, we can keep using `#define` but we should use the same `#define` everywhere in the codebase and not just here.